v3/src/components/graph/adornments/movable-value.tsx

```diff
@@ -29,24 +29,22 @@ export const MovableValue = (props: {
 
     [line, cover].forEach(aLine => {
       aLine
-        // .attr('transform', transform)
-        .attr('x1', x(value))
+        .attr('x1', xScale(value))
         .attr('y1', top)
-        .attr('x2', x(value))
+        .attr('x2', xScale(value))
         .attr('y2', bottom)
     })
-  }, [bottom, top, valueObject, x])
+  }, [bottom, top, valueObject, xScale])
 
   const refreshValueLabel = useCallback((value: number) => {
-    const leftEdge = valueRef.current?.parentElement?.getBoundingClientRect().left,
-      screenX = x(value) + (leftEdge || 0),
-      screenY = Number(valueRef.current?.getBoundingClientRect().top) - 12,
+    const leftEdge = margin.left,/*valueRef.current?.parentElement?.getBoundingClientRect().left,*/
```

Comment:
Do we still need the comment?
```suggestion
    const leftEdge = margin.left,
```